test/integration/client-side-encryption/client_side_encryption.spec.test.js

```diff
@@ -7,6 +7,52 @@ const {
   generateTopologyTests
 } = require('../../tools/spec-runner');
 
+const isAuthEnabled = process.env.AUTH === 'auth';
+
+// 'TODO: NODE-3891 - fix tests broken when AUTH enabled'
+const skippedAuthTests = [
+  'Insert a document with auto encryption using the AWS provider with temporary credentials',
+  'Insert a document with auto encryption using Azure KMS provider',
+  '$rename works if target value has same encryption options',
+  'Insert with deterministic encryption, then find it',
+  'Insert with randomized encryption, then find it',
+  'Bulk write with encryption',
+  'Insert with bypassAutoEncryption',
+  'Insert with bypassAutoEncryption for local schema',
+  'ping is bypassed',
+  'deleteOne with deterministic encryption',
+  'deleteMany with deterministic encryption',
+  'distinct with deterministic encryption',
+  'Explain a find with deterministic encryption',
+  'Find with deterministic encryption',
+  'Find with $in with deterministic encryption',
+  'findOneAndReplace with deterministic encryption',
+  'findOneAndUpdate with deterministic encryption',
+  'Insert a document with auto encryption using GCP KMS provider',
+  'getMore with encryption',
+  'unset works with an encrypted field',
+  'updateOne with deterministic encryption',
+  'updateMany with deterministic encryption',
+  'type=date',
+  'type=regex',
+  'type=timestamp',
+  'type=javascript',
+  'type=binData',
+  'type=int',
+  'type=objectId',
```

Comment:
We wanted to go back and reassess if these tests would get fixed by the sharded uri fix we had to make

Reply:
<img width="554" alt="Screen Shot 2022-02-02 at 2 37 04 PM" src="https://user-images.githubusercontent.com/23407842/152224367-c56f0907-c33a-434e-86a9-ec8a93f2b4ac.png">

Our fix to the test runner didn't fix these tests.  These tests seem to be failing with an "extra ping command".  I can dig into these and see if I can find a fix but I'd rather wait and fix them as a part of the follow up ticket

Reply:
Ah ok yea at least we confirmed its not about the content of the test, I'm fine with defering